### PR TITLE
Remove unused CSS, simplify entrypoints and update README

### PR DESCRIPTION
**Pull request by lyzadanger**
This PR removes unused legacy `frontend-shared` CSS from this project, and simplifies remaining external (S)CSS. **The size of the generated `frontend_apps` stylesheet is reduced by about 45%.** 🎉 

Some last lingering local utility classes are removed here. Through experience I have come to believe (strongly, I'll die on this hill) that the cost of abstraction is higher than the cost of slightly-wet Tailwind utility classes that are right in front of your face.

### CODE REVIEW

File: lms/static/styles/frontend_apps.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // CSS Entry point for front-end (preact) app
 @use 'base';
 @use 'components';
-@use 'utilities';
+@use 'tailwindcss/utilities';
```
Comment (lyzadanger) [inline lms/static/styles/frontend_apps.scss:4]: The entire `utilities` layer is just Tailwind's utilities layer. We still add a couple of things to the `base` and `components` layers. There is a SASS limitation that `@use` directives have to come first in a module, so `base` and `components`, while they are very small modules, need to remain external. i.e. we can't have something like this in a single module:

```scss
@use 'tailwindcss/base';

// Element styles
@layer base {
  html,
  body {
    height: 100%;
  }
  // ...
}

@use 'tailwindcss/components`;

@layer components {
  // A warning banner shown at the top of the view in unsupported browsers.
  // This is styled to match warning toasts and banners in the Hypothesis client.
  .browser-check-warning {
    // ...
  }
}

//...

@use 'tailwindcss/utilities';
```

even though, once processed, that's the net result. The layers need to be "activated" in order, and we can't split up `@use` directives.

File: lms/static/styles/ui-playground.scss [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-@use 'base';
-@use 'components';
-@use 'utilities';
+@use 'frontend_apps';
 
 // Include the CSS for the Library components themselves
 @use '@hypothesis/frontend-shared/styles/library';
 
+// TODO: Obviate the need for these overrides/customizations
```
Comment (lyzadanger) [inline lms/static/styles/ui-playground.scss:9]: There's a separate small project here.

File: lms/static/styles/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # CSS Styling for LMS
 
-There are currently four CSS stylesheets generated from CSS and SASS source here:
+There are currently three CSS stylesheets generated from source here:
 
-- `frontend_apps.css`, entry point `frontend_apps.scss`: The CSS to style the (Preact) front-end components. Components have been converted to use Tailwind utility-first styling in most cases.
-- `ui-playground.css`, entry point `ui-playground.scss`: Generates styles needed to style the Pattern Library. Currently in flux; mostly converted to Tailwind but with some extra local override classes.
-- `lms.css`, entry point `lms.scss`: The styles output from here are used by back-end interfaces (see `lms/templates/base.html.jinja2`). These styles should be reviewed at some point and evaluated for conversion to Tailwind, if feasible.
+- `frontend_apps`: The CSS to style the (front-end) LMS application
+- `lms`: Used by LMS app (back-end) views (see `lms/templates/base.html.jinja2`). These styles should be reviewed at some point and evaluated for conversion to Tailwind, if feasible.
```
Comment (lyzadanger) [inline lms/static/styles/README.md:6]: This is a separate project: these styles are definitely used in pyramid/jinja views. But are all of them used? Could we update those views to use more conventional CSS/Tailwind? Etc.

**Review by acelaya — APPROVED** [review]
👍🏼

**Pull request merged**